kalite/settings/base.py

```diff
@@ -322,6 +323,9 @@
 
 ROOT_URLCONF = "kalite.distributed.urls"
 
+BACKUP_DIRPATH = os.path.join(expanduser("~"), 'ka-lite-backups')
```

Comment:
true, I concede... it might not make sense to encourage backing up in the source dir of some of the files that are backed up.

In the end, we should encourage/require people to manually set the backup path.